src/main/java/org/opensrp/repository/postgres/mapper/xml/ProductCatalogueMapper.xml

```diff
@@ -9,7 +9,7 @@
     -->
     <id column="unique_id" jdbcType="BIGINT" property="uniqueId" />
     <result column="product_name" jdbcType="VARCHAR" property="productName" />
-    <result column="json" jdbcType="OTHER" property="json" typeHandler="org.opensrp.repository.postgres.handler.ProductCatalogueTypeHandler" />
+    <result column="product_catalogue_json" jdbcType="OTHER" property="json" typeHandler="org.opensrp.repository.postgres.handler.ProductCatalogueTypeHandler" />
```

Comment:
Can you just create a custom result for query that returns both stock and catalog

Reply:
Yes, a custom query was made which returns both stock and product catalogue details

Reply:
Yes, a custom query was made which returns both stock and product catalog details